node/src/components/rpc_server/rpcs/info.rs

```diff
@@ -47,10 +47,10 @@ static GET_PEERS_RESULT: Lazy<GetPeersResult> = Lazy::new(|| GetPeersResult {
 static GET_VALIDATOR_CHANGES_RESULT: Lazy<GetValidatorChangesResult> = Lazy::new(|| {
     let era_changes = JsonEraChange::new(EraId::new(1), ValidatorChange::Added);
     let public_key = PublicKey::doc_example().clone();
-    let validator_info = vec![JsonValidatorInfo::new(public_key, vec![era_changes])];
+    let validator_info = vec![JsonValidatorChanges::new(public_key, vec![era_changes])];
```

Comment:
We could call this `changes` as well.
```suggestion
    let changes = vec![JsonValidatorChanges::new(public_key, vec![era_changes])];
```

Reply:
Addressed in [`d0417f9`](https://github.com/darthsiroftardis/casper-node/pull/12/commits/d0417f96c45978441e9e0c0e615610862fca246c).